Makefile

```diff
@@ -13,8 +18,8 @@ build-deps: .build-deps
 #	GO111MODULE=off go get -u github.com/google/addlicense
 
 deps: .deps
-.deps:
-	@if [ -d extractor ]; then cd extractor && npm install; fi
+# seeL https://stackoverflow.com/a/59272238 for explination of if block.
```

Comment:
dang it..

Reply:
oh... yeah. Eventually, I shouldn't need the phony target here. Once I get the paths between modules PR merged, I'm planning on moving `deps`, `extractor`, `gateway`, `indexer`, and `tracker` all under a single `services` directory. But for now... :-/

Reply:
Makes sense. the Dang it was specifically about the errant `L`...